netlify.toml

```diff
@@ -33,4 +33,21 @@ ID = "48a8adb4-6837-44ba-a9c4-267b56de8212"
 # Deploy Preview context: all deploys resulting from a pull/merge request will
 # inherit these settings.
 # [context.deploy-preview]
-  # publish = "dist/"
\ No newline at end of file
+  # publish = "dist/"
+
+[[redirects]]
+from = "/features/message-pumps/service-bus"
+to = "/features/message-handling/service-bus"
+
+[[redirects]]
+from = "/features/message-pumps/customization"
+to = "/features/message-handling/customize-general"
+
+[[redirects]]
+from = "/features/message-pumps/service-bus-azure-functions"
+to = "/features/message-handling/service-bus-azure-functions"
+
+[[redirects]]
+from = "/features/service-bus"
+to = "/features/service-bus-extensions"
```

Comment:
Had to add some redirects as we have reworked the message handling structure:
https://github.com/arcus-azure/arcus.messaging/pull/268/files